src/components/InventoryPage/InventoryPage.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Table,
+  Thead,
+  Tbody,
+  Button,
+  Tr,
+  Td,
+  Th,
+  TableContainer,
+  useDisclosure,
+  Text,
+} from '@chakra-ui/react';
+import './InventoryPage.module.css';
+
+import DonationModal from './DonationModal';
+import { getDonationsFromDB } from '../../utils/InventoryUntils';
+
+const InventoryPage = () => {
+  const [users, setUsers] = useState([]);
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const [donationData, setDonationData] = useState({});
+
+  const handleRowClick = data => {
+    setDonationData(data);
+    onOpen();
+  };
+
+  function makeStatus(newStatus) {
+    if (newStatus === 'denied') {
+      return (
+        <Button size="xs" colorScheme="red">
+          REJECTED
+        </Button>
+      );
+    }
+    if (newStatus === 'approved') {
+      return (
+        <Button size="xs" colorScheme="green">
+          APPROVED
+        </Button>
+      );
+    }
+    if (newStatus === 'flagged') {
+      return (
+        <Button size="xs" colorScheme="gray">
+          FLAGGED
+        </Button>
+      );
+    }
+    return (
+      <Button size="xs" colorScheme="gray">
+        {newStatus}
+      </Button>
+    );
+  }
+
+  function makeDate(dateDB) {
+    const d = new Date(dateDB);
+    return `${d.getMonth() + 1}/${d.getDate()}/${d.getFullYear()}`;
+  }
```

Comment:
Since this is being used in two places, you can move this to a utils file to import instead.